src/workflow/workflow.js

```diff
@@ -237,7 +256,11 @@ const pinAndSetImageCid = async (wfConfig) => {
       metaCidColumn.records.push('');
     }
 
-    if (i >= startRecordNo && i < endRecordNo && !metaCidColumn.records[i]) {
+    if (i >= startRecordNo && i < endRecordNo) {
+      if (metaCidColumn.records[i]) {
+        console.log(`metadata for the row #${i} is already set, skipping`);
```

Comment:
"set" can be confusing since there is another step to set metadata on chian. this can be called "generated" or "uploaded"?!